data/sql/updates/pending_db_world/rev_1608949912835926000.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1608949912835926000');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` IN (19257, 19258);
+
+-- Arcanist Torseldori
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 19257;
```

Comment:
ALWAYS specify the `source_type` as well when deleting `smart_scripts`, otherwise you will delete also the `entryorguid` of other types, which is not intended :)